packages/metrics/src/features/metrics/utils.ts

```diff
@@ -268,7 +268,7 @@ export const fetchEstimateForTargetDaysByLocationId = async (
   timeTo: string
 ): Promise<IEstimation> => {
   if (locationId) {
-    const locationData: Location = await fetchFHIR(locationId, authHeader)
+    const locationData: Location = await fetchFHIR(locationId)
```

Comment:
Only token removals in this file

![](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExeHVjbm1wM3prc3dkZm01a2JtdnA4eTV4ZjRsNHBqcXB3NnhnbmIzZyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9dg/BGVAiLhU93r0uAl4Qj/giphy.gif)